benchmarks/helics/EchoMessage.hpp

```diff
@@ -84,72 +83,72 @@ class EchoMessageLeaf
     helics::Endpoint ept;
 
-    int index_ = 0;
+    int index_       = 0;
     bool initialized = false;
-    bool readyToRun = false;
+    bool readyToRun  = false;
 
   public:
-    EchoMessageLeaf () = default;
+    EchoMessageLeaf() = default;
 
-    void run (std::function<void()> callOnReady = {})
+    void run(std::function<void()> callOnReady = {})
     {
         if (!readyToRun)
         {
-            makeReady ();
+            makeReady();
         }
         if (callOnReady)
         {
-            callOnReady ();
+            callOnReady();
         }
-        mainLoop ();
+        mainLoop();
     };
-    void initialize (const std::string &coreName, int index)
+    void initialize(const std::string & coreName, int index)
     {
-        std::string name = "echoleaf_" + std::to_string (index);
-        index_ = index;
+        std::string name = "echoleaf_" + std::to_string(index);
+        index_           = index;
         helics::FederateInfo fi;
         fi.coreName = coreName;
-        mFed = std::make_unique<helics::MessageFederate> (name, fi);
+        mFed        = std::make_unique<helics::MessageFederate>(name, fi);
         // this is a local endpoint
-        ept = mFed->registerEndpoint ("leaf");
+        ept         = mFed->registerEndpoint("leaf");
         initialized = true;
     }
 
-    void makeReady ()
+    void makeReady()
     {
         if (!initialized)
         {
-            throw ("must initialize first");
+            throw("must initialize first");
         }
-        mFed->enterExecutingMode ();
+        mFed->enterExecutingMode();
         readyToRun = true;
     }
 
-    void mainLoop ()
+    void mainLoop()
     {
         int cnt = 0;
         // this is  to make a fixed size string that is different for each federate but has sufficient length to
         // get beyond SSO
-        const std::string txstring = std::to_string (100000 + index_) + std::string (100, '1');
-        const int iter = 5000;
+        const std::string txstring = std::to_string(100000 + index_) + std::string(100, '1');
+        const int iter             = 5000;
         while (cnt <= iter + 1)
         {
-            mFed->requestNextStep ();
+            mFed->requestNextStep();
             ++cnt;
             if (cnt <= iter)
             {
-                ept.send ("echo", txstring);
+                ept.send("echo", txstring);
             }
-            while (ept.hasMessage ())
+            while (ept.hasMessage())
             {
-                auto m = ept.getMessage ();
-                auto &nstring = m->data.to_string ();
+                auto m         = ept.getMessage();
+                auto & nstring = m->data.to_string();
                 if (nstring != txstring)
                 {
-                    throw( "incorrect string");
+                    throw("incorrect string");
                 }
             }
         }
-        mFed->finalize ();
+        mFed->finalize();
     }
 };
```

Comment:
Is there a way to make clang format add a newline at the end of files if not present?

Reply:
I think there is not.